apturicovid/viewController/stats collection view/StatsFooterView.swift

```diff
@@ -1,33 +1,23 @@
 import UIKit
+import SafariServices
 
 //MARK: - StatsHeaderView
 class StatsFooterView: UICollectionReusableView {
 
     @IBOutlet weak var linkButton: AutoLayoutButton!
     
     @IBAction func linkTap(_ sender: Any) {
-        openUrlLink()
+        openURL()
     }
     
     static var identifier: String { String(describing: self) }
-    var linkUrl: URL?
+    var openURL: (() -> Void)!
```

Comment:
+ like the idea about just keeping a lambda here, so the logic what happens on the button press is left to the VC
- would prefer `?` instead of `!` here, because of a developer error the `setup` method might not be called yet and thus the value can be `nil`